contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1749,9 +1749,21 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		return;
 
 	type_def = defGetTypeName(as_type);
+	type_names = type_def->names;
+	switch (list_length(type_def->names))
+    {
+        case 2:
+            strVal(linitial(type_names)) = get_physical_schema_name(get_cur_db_name(),strVal(linitial(type_names)));
+            break;
+        case 3:
+            strVal(lsecond(type_names)) = get_physical_schema_name(strVal(linitial(type_names)),strVal(lsecond(type_names)));
+            break;
+    }
+	*newtypid = typenameTypeId(pstate, type_def);
```

Comment:
Fix indentation.
Why it was working earlier without this change? Is it because earlier it wasn't getting user defined datatypes?